packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  settings as ddsSettings,
+  markdownToHtml,
+} from '@carbon/ibmdotcom-utilities';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * ContentArrayBlock Component, for use with content arrays
+ *
+ * @param {object} props props object
+ * @param {string} props.heading  Heading objects
+ * @param {copy} props.copy Heading copy copy
+ * @param {*} props.children JSX Components
+ * @param {*} props.cta JSX Components
+ * @returns {*} JSX ContentArrayBlock component
+ */
+const ContentBlock = ({ heading, copy, children, cta, customClassName }) => {
+  const className = cx(`${prefix}--content-block`, customClassName);
+
+  return (
+    <div data-autoid={`${stablePrefix}--content-block`} className={className}>
+      {heading && (
+        <h2
+          data-autoid={`${stablePrefix}--content-block__title`}
+          className={`${prefix}--content-block__title`}>
+          {heading}
+        </h2>
+      )}
+      {copy && (
+        <div
+          className={`${prefix}--content-block__intro`}
+          dangerouslySetInnerHTML={{ __html: markdownToHtml(copy) }}
+        />
+      )}
+      <div data-autoid={`${stablePrefix}--content-block__children`}>
+        {children}
+      </div>
+      {cta && (
+        <div
+          data-autoid={`${stablePrefix}--content-block__cta`}
+          className={cx(`${prefix}--content-block__cta`)}>
+          {cta}
+        </div>
```

Comment:
This should pull in the `CTA` component that @mkothur is creating.

Reply:
Ah, gotcha. I didn't realize there was a new `CTA` component when I started this.